src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/VisitSessionDto.kt

```diff
@@ -37,4 +58,7 @@ data class VisitSessionDto(
   @Schema(description = "The end timestamp for this visit session", example = "1", required = true)
   val endTimestamp: LocalDateTime,
 
+  @Schema(description = "The day of week for the visit session", example = "MONDAY", required = true)
```

Comment:
Is `dayOfWeek` actually needed on this as for an individual visit session it should be what's in `startTimestamp`? Thinking that in the front-end when we retrieve the list of visit sessions to display we show day and date for each so would be using the timestamp rather than this.

Reply:
It is true, you don't need it, does add anything readability wise/ or for logging? your call?

Reply:
Are individual visit sessions MVP?

Reply:
I think the example value of 1 is misleading here - 1 converted to a timestamp is a few thousand years BC. It would be better to use a more realistic value - as a timestamp e.g. "23-May-2023 11:00:00" - in correct format of course.

Reply:
It is totally misleading.. for a start its not a timestamp... doc is wrong also,.. not really part of review but will change doc, but not the val name... that will have to wait

Reply:
> It is true, you don't need it, does add anything readability wise/ or for logging? your call?

I'd say probably not worth it at the moment as - at least from front-end app point of view.

Reply:
Cool, I will take it out then!